js/react/rules/react-a11y.js

```diff
@@ -40,27 +44,25 @@ module.exports = {
     }],
 
     // disallow href "#"
-    // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/href-no-hash.md
-    'jsx-a11y/href-no-hash': [2, {
-      components: ['Link', 'Anchor', 'a']
+    // https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md
+    'jsx-a11y/anchor-is-valid': [2, {
+      components: ['Link', 'Anchor', 'a'],
+      aspects: ['invalidHref'],
```

Comment:
If we leave off aspects, it will throw anytime we want to use a button instead of a link.  I wasn't sure if these instances were something we wished to stop or not

Reply:
I think we should still force this - if it's a `Button` we should still have an href. If it's truly _just_ a `Button` then we should be using `Button` and the `onClick` prop.